Answers.md

```diff
@@ -1,5 +1,13 @@
 1.  Explain the differences between `client-side routing` and `server-side routing`.
-1.  What does HTTP stand for?
+Both of these refer mainly to how we access data that will be rendered in the browser. Server-side routing is done when we need to contact a server directly and access data that it will send back to us to be rendered. Client-side routing is referring to data that is stored within the browser in order to find what we need to render. 
```

Comment:
The data can be stored in the server in both cases, but the updating from the routes is handled by javascript and ajax calls in client-side routing, without reloading, whereas server side the whole page change is sent up as new html